src/shared/firewall-util-nft.c

```diff
@@ -34,7 +34,26 @@
 
 #define UDP_DPORT_OFFSET 2
 
-static sd_netlink_message **netlink_message_unref_many(sd_netlink_message **m) {
+/* for testing */
+static const char* nft_table_name(void) {
+        static const char *cached;
```

Comment:
doesn't matter, but i would always initialize such internal vars to NULL implicitly. but i am aware others disagree, and doesn' matter